topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+---
+layout: tutorial_hands_on
+
+title: Evaluating Reference Data for Bulk RNA Deconvolution
+subtopic: deconvo
+priority: 3

```

Comment:
did you mean for this tutorial to be 3rd in the subsection? It is second now because the other tutorials in the subsections have priorities 1 and 4 listed. So please double check if the order is how you want it now.